includes/in-person-payments/templates/html-in-person-payment-receipt.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * In Person Payments Receipt Template
+ *
+ * @package WooCommerce\Payments
+ */
+
+/**
+ * Helper to generate markup to render a price.
+ *
+ * @param  array  $product The product to display.
+ * @param  string $currency The currency to display.
+ * @return string
+ */
+function format_price_helper( array $product, string $currency ): string {
+	$active_price  = $product['price'];
+	$regular_price = $product['regular_price'];
+	$has_discount  = $active_price !== $regular_price;
+
+	if ( $has_discount ) {
+		return '<s>' . wc_price( $regular_price, [ 'currency' => $currency ] ) . '</s> ' . wc_price( $active_price, [ 'currency' => $currency ] );
+	}
+
+	return wc_price( $active_price, [ 'currency' => $currency ] );
+}
+
+?><!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Print Receipt</title>
+	<style>
+		body {
+			margin: 0;
+			padding: 0;
+			border: 0;
+		}
+
+		.align-left {
+			text-align: left;
+		}
+
+		.align-right {
+			text-align: right;
+		}
+		.align-top {
+			vertical-align: top;
+		}
+
+		.receipt {
+			min-width: 130px;
+			max-width: 300px;
+			margin: 0 auto;
+			text-align: center;
+			font-family: SF Pro Text, sans-serif;
```

Comment:
"SF Pro Text" is an Apple font, is it really available on the thermal printer system? Maybe we can provide a choice of fonts or at least embed the one which we recommend?

Reply:
I see your point. I believe that `sans-serif` is a safe callback in case SF Pro Text is not available. 
This seems to work according to the tests we did with the thermal printer